cs/di-defaultextensions.texy

```diff
@@ -0,0 +1,86 @@
+DI: Defaultní rozšíření pro kontejner
+*************************************
+
+Nette obsahuje několik předdefinovaných [Nette\DI\CompilerExtension |api:] pro DI kontejner.
+
+ExtensionsExtension
+===================
+
+Díky tomuto rozšíření můžeme zaregistrovat další rozšiření pomocí konfiguračních NEON souborů.
+
+/---neon
+extensions:
+	blog: App\DI\BlogExtension
+	user: App\DI\UserExtension
+\---
+
+InjectExtension
+===============
+
+Pokud u služeb nastavíme příznak `inject: true`, aktivujeme tím předávání závislostí přes veřejné
+proměnné s anotací *@inject*.
+
+/---neon
+services:
+	blog.articles:
+		class: App\Model\Articles
+		inject: true
+\---
+
+/---php
+namespace App\Model;
+
+use Nette\Database\Connection;
+
+class Articles
+{
+
+	/** @var Connection @inject */
+	public $connection;
+
+}
+\---
+
+V základním nastavení je toto rozšíření aktivováno pouze pro [Nette\Application\UI\Presenter |api:].
```

Comment:
Tomu rozumim, ale tady je odkaz primo na tridu o ktere se mluvi ne? Nechci to rozebirat, klidne ho udelam, spis abych vedel do budoucna.